src/applications/dhp-connected-devices/components/UnauthenticatedPageContent.jsx

```diff
@@ -16,10 +21,23 @@ export const UnauthenticatedPageContent = () => {
           If you don’t have any of these accounts, you can create a free ID.me
           account now.
         </div>
-        <div className="dhp-login-button">
-          <UserNav isHeaderV2 />
-        </div>
+        <button type="button" className="usa-button" onClick={onSignInClicked}>
+          Sign in or create an account
+        </button>
       </va-alert>
     </>
   );
 };
+
+const mapDispatchToProps = dispatch => ({
+  toggleLoginModal: open => dispatch(toggleLoginModalAction(open)),
```

Comment:
This is just a suggestion, but since you know that you are only going to be using `true` as the argument for this function, you could simply write this as:
```js
toggleLoginModal: () => dispatch(toggleLoginModalAction(true)),
```

This would allow you to replace L20 above with:
```js
<button type="button" className="usa-button" onClick={toggleLoginModal}>
```
and you would could then remove `onSignInClicked`